backend/migrations/20211219190858-init_collections.js

```diff
@@ -0,0 +1,67 @@
+const bcrypt = require('bcrypt');
+
+const saltRounds = 10;
+
+module.exports = {
+  async up(db, client) {
+    DEFAULT_USERS_DOCS.map(async (doc) => {
+      const salt = bcrypt.genSaltSync(saltRounds);
+      doc.password = bcrypt.hashSync(doc.password, salt);
+      await db.collection('users').insertOne(doc);
+    });
+    // await db.collection('skills');
+    // await db.collection('courses');
+    // await db.collection('userSkills');
+    // await db.collection('userCourses');
+  },
+
+  async down(db, client) {
+    await db.collection('users').drop();
+    // await db.collection('skills').drop();
+    // await db.collection('courses').drop();
+    // await db.collection('userSkills').drop();
+    // await db.collection('userCourses').drop();
+  },
+};
+
+const DEFAULT_USERS_DOCS = [
+  {
+    name: 'admin',
+    password: 'admin',
```

Comment:
We can't store password as it is.
Hash please this password and store it in `passwordHash` field.

Reply:
It's being hashed during proccess of migrating up. Or you want password to be stored as hash right in `DEFAULT_USER_DOCS`?

Reply:
Oh, you're right!
Let's leave it in such way